### PR TITLE
FIX: Added University of Washington to institutions db

### LINKED ISSUE #2: institutions db needs entry for University of Washington

### PR DESCRIPTION
**Pull request by eaculb**
Closes #2

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -149,3 +149,11 @@ ucopenhagen:
   name: University of Copenhagen
   state: ''
   zip: ''
+uwashington:
+  aka: [University of Washington]
```
Comment (sbillinge) [inline db/institutions.yml]: Isn't UW known by any other names?  The Bulldogs or something?

Comment (sbanerjee23) [reply db/institutions.yml]: It's the UW Huskies :dog2:

Comment (eaculb) [reply db/institutions.yml]: The Huskies are a mascot, not an AKA, so I am opting not to add that.

Comment (eaculb) [reply db/institutions.yml]: Perhaps UW, though?

Comment (sbillinge) [reply db/institutions.yml]: OK, no problem.  But what about UW?

**Pull request merged**